pytorch_lightning/loops/base.py

```diff
@@ -88,11 +102,7 @@ def run(self, *args: Any, **kwargs: Any) -> Optional[Any]:
         if self.skip:
             return self.on_skip()
 
-        if self.restarting:
```

Comment:
So why do we not want the `restarting` property anymore?

And why did you leave it in the `__init__`?

Reply:
This is still used in reset `self.restarting`. Check https://github.com/PyTorchLightning/pytorch-lightning/pull/8364.

Adrian convinced me it was cleaner to make it manual.

self.restarting is automatically set to True by the Trainer when calling loop `load_state_dict` function.

Reply:
Where exactly is that PR setting `restarting` on the loops?

Reply:
Hey @carmocca, right there: https://github.com/PyTorchLightning/pytorch-lightning/pull/8334/files#diff-4076755145b42e55247c8c12a7a7961a91f4d28a3e962c80a98161163bfa3ec9R205.

Each loop is going to load its own checkpoint and be put in restarting mode.